emcc.py

```diff
@@ -111,7 +111,11 @@
 
 DEFAULT_ASYNCIFY_EXPORTS = [
   'main',
-  '__main_argc_argv'
+  '__main_argc_argv',
+  # Embind's async template wrapper functions. These functions are usually in
+  # the function pointer table and not called from exports, but we need to name
+  # them so the JSPI pass can find and convert them.
+  '_ZN10emscripten8internal5async*'
```

Comment:
I'm not that familiar with embind internals, can you tell me where these functions are created? (I looked in `system/lib/embind/bind.cpp` but it seems that's not the place)

Reply:
There are several Invoker structs e.g. Invoker, FunctorInvoker, MethodInvoker https://github.com/emscripten-core/emscripten/blob/bf1a13252e36755e078b63b05358a564bd0091e4/system/include/emscripten/bind.h#L363-L364
The wrapper around the Invokers in this patch comes from: https://github.com/brendandahl/emscripten/blob/23ad23a4e595ab0d7835a95cdfc0ef1cd6b85037/system/include/emscripten/bind.h#L424-L430

Reply:
I see, thanks. That's in the C++ parts I understand less, then :smile: